releasenotes/notes/2023.1-update-packages-and-containers-q2-2024-02ed375b9c73ac75.yaml

```diff
@@ -0,0 +1,41 @@
+---
+upgrade:
```

Comment:
Upgrade is for things the user should be aware of when they do an upgrade e.g. a warning if something is not backwards-compatible.
These lines should be features instead.
It would also be nice to have the version we are upgrading from, but if it is difficult to find that information then don't worry about it

Reply:
Got it. I'll change it to features.
Do you mean the previous versions of prometheus, grafana, opensearch containers?

Reply:
That's possible. I'll include that too.